backend/database/schema.sql

```diff
@@ -0,0 +1,110 @@
+create type role_type as enum ('admin', 'member');
+create type colour_type as enum ('red', 'orange', 'yellow', 'blue', 'green', 'grey', 'brown', 'purple', 'pink');
+create type checkout_type as enum ('borrow', 'use');
+create type checkout_status as enum ('waiting', 'checked_out', 'returned');
+create type approval_status as enum ('pending_approval', 'approved', 'not_approved');
+
+CREATE TABLE person (
+    zid             varchar(7) PRIMARY KEY,
+    password        varchar(12) NOT NULL,
+    first_name      varchar(30) NOT NULL,
+    last_name       varchar(30),
```

Comment:
Perhaps consider using `text` type for some of these (particularly in emails and names)?
`text` provides variable unlimited length while `varchar(n)` has a limit of `n` characters

Reply:
Agreed